workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/main.tsx

```diff
@@ -3,26 +3,103 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {ClayVerticalNav} from '@clayui/nav';
+import {useState} from 'react';
 import {createRoot} from 'react-dom/client';
 import {QueryClient, QueryClientProvider} from 'react-query';
+import {HashRouter, Route, Routes} from 'react-router-dom';
 
 import TicketApp from './pages/TicketApp';
+import TicketsByStatusDashboard from './pages/TicketsByStatusDashboard';
+import {Liferay} from './services/liferay';
 
+import './styles/Main.css';
+export const SPRITEMAP =
+	(Liferay as any).ThemeDisplay.getPathThemeImages() + '/clay/icons.svg';
+
+const DEFAULT_ROUTE: string = '#dashboard';
 const queryClient = new QueryClient();
-class TicketWebComponent extends HTMLElement {
+
+const Main: React.FC = () => {
+	const [activeItem, setActiveItem] = useState('1');
+
+	if (window.location.href.indexOf('#') === -1) {
+		window.location.href = window.location.href + DEFAULT_ROUTE;
+	}
+
+	function navItemClicked(item: any) {
+		setActiveItem(item.id);
+	}
+
+	return (
+		<QueryClientProvider client={queryClient}>
+			<section className="container-fluid current-tickets m-0 p-0 row">
```

Comment:
We also probably don't need the `section` element or the `current-tickets` class.

Reply:
Why not using bootstrap rows and columns? it seems a bit complicated to use the Layout component.

Reply:
The layout component isn't that hard to use. Let me know if you have any specific questions about it's usage. It just provides an api for applying the bootstrap classes so that you're not manually adding the classes. This is better for maintainability, provides TypeScript support, and provides a layer that we can change without needing to make changes to your code directly.